src/frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "buildOverlay": "npm run build -w overlay",
     "buildAdmin": "npm run build -w admin",
+    "buildLocatorAdmin": "npm run build -w admin",
```

Comment:
This is the same as the previous line. Why is it needed?

Reply:
As I undestood, it's needed becuase gralde task name is used to make a task. 
You can use         `npmCommand = listOf("run", "buildAdmin")` inside task description instead.